pkg/services/sqlstore/alert_test.go

```diff
@@ -84,6 +84,12 @@ func TestAlertingDataAccess(t *testing.T) {
 					So(err, ShouldNotBeNil)
 				})
 
+				Convey("alert is paused", func() {
+					alert, _ = getAlertById(1)
+					currentState := alert.State
+					So(currentState, ShouldEqual, "paused")
+				})
+
```

Comment:
Thank you for contributing this!
I will definitely merge it after resolving the time problem!